public/Query.js

```diff
@@ -769,4 +771,31 @@ Query.executeFromSearch = function () {
 
 window.addEventListener('popstate', function () {
 	Query.executeFromSearch()
-})
\ No newline at end of file
+})
+
+/* Make table header fixed after scroll limit is reached */
+function fixedHeader() {
+
+	document.getElementById('fixed-table-header').innerHTML = ''
+
+	var headers = document.getElementsByClassName('header')
+	headers = Array.prototype.slice.call(headers)
+	var newTable = document.createElement('table')
```

Comment:
Use [`Panel.create`](https://github.com/clubedaentrega/node-mongo-admin/blob/master/public/Panel.js#L36) instead of `document.createElement`